src/components/topbar/TryVueNodeBanner.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div
+    v-if="showVueNodesBannerRef"
+    class="pointer-events-auto w-full h-10 bg-gradient-to-r from-blue-600 to-blue-700 flex items-center justify-between px-4"
+  >
+    <div class="w-5 h-5"></div>
+    <div class="flex items-center">
+      <i class="icon-[lucide--sparkles]"></i>
+      <h5 class="pl-2">{{ $t('vueNodesBanner.message') }}</h5>
```

Comment:
nit: I don't like using out-of-order headings, it's an accessibility issue.
https://www.a11yproject.com/posts/how-to-accessible-heading-structure/